channeldb/migration_01_to_11/graph.go

```diff
@@ -744,7 +744,7 @@ func deserializeLightningNode(r io.Reader) (LightningNode, error) {
 		return LightningNode{}, err
 	}
 
-	fv := lnwire.NewFeatureVector(nil, lnwire.GlobalFeatures)
+	fv := lnwire.NewFeatureVector(nil, lnwire.Features)
```

Comment:
(maybe this was already discussed but the diff is gone) Shouldn't this be isolated?

Reply:
also fixed, replaced w/ nil